android/src/main/java/com/oktareactnative/OktaSdkBridgeModule.java

```diff
@@ -595,4 +599,17 @@ public void onError(String e, AuthorizationException error) {
             promise.reject(OktaSdkError.OKTA_OIDC_ERROR.getErrorCode(), e.getLocalizedMessage(), e);
         }
     }
+
+    private boolean isIdTokenNotExpired(@Nullable String idToken) {
+        if (idToken == null) {
+            return false;
+        }
+        try {
+            OktaIdToken oktaIdToken = OktaIdToken.parseIdToken(idToken);
+            long nowInSeconds = System.currentTimeMillis() / 1000L;
```

Comment:
@aarongranick-okta do we do checks like this in other places? Do we need to ensure the clock is up to date given something like NTP?